brozzler/cli.py

```diff
@@ -371,11 +417,11 @@ def on_screenshot(screenshot_jpeg):
             on_screenshot=on_screenshot,
             enable_youtube_dl=not args.skip_youtube_dl,
         )
-        logging.info("outlinks: \n\t%s", "\n\t".join(sorted(outlinks)))
+        logger.info("outlinks: \n\t%s", "\n\t".join(sorted(outlinks)))
```

Comment:
Consider removing newlines and probably all escaped whitespace chars from log statements. You shouldn't need them with structured output and newlines in statements will probably make log aggregation harder and/or less effective

Reply:
Good call; did that here and in the two tests that did the same thing.

```
2025-02-22 00:14:00 [info     ] outlinks                       [brozzler.cli.brozzle_page(cli.py:420)] outlinks=['https://www.iana.org/domains/example']
```